src/containers/ViewerPanel/index.tsx

```diff
@@ -200,8 +204,24 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
     }
 
     public componentDidUpdate(prevProps: ViewerPanelProps) {
-        const { error } = this.props;
+        const { error, isFullScreen } = this.props;
```

Comment:
Hm, ok yes, this used to be global because it was how I controlled side panel rendering, but you streamlined that so I guess it isn't needed anymore.

That said, side panels come and go based on size/fullscreen during embed, but are always rendering in the main viewer. If I remove the `.embed` className in `SideBar` I can make the side panels also conditional on a large enough window size, which makes sense to me since the other changes to playback controls are also propagating to all windows.

I'm going to include it, but lmk if you don't think that's correct to do?